.circleci/config.yml

```diff
@@ -222,6 +222,12 @@ commands:
                   if [[ $CIRCLE_TAG == android-v* ||  $CIRCLE_BRANCH == main ]]; then
                     make sdkRegistryUpload
                   fi
+                  if [[ $CIRCLE_TAG == android-v* ]]; then
+                    git config --global user.email "MapboxCI@users.noreply.github.com"
+                    git config --global user.name "MapboxCI"
+                    export GITHUB_TOKEN=$(./mbx-ci github writer token)
+                    make sdkRegistryPublish
+                  fi
```

Comment:
@Chaoba could you add this reference CI integration to the documentation of sdk-regitstry plugin?

Reply:
Ok